.github/workflows/ci_benchmark.yml

```diff
@@ -73,7 +73,7 @@ jobs:
           set -x
           python -m pip install asv virtualenv packaging
 
-          git clone -b main https://github.com/astropy/astropy-benchmarks.git --single-branch
+          git clone -b fix/asv_conf https://github.com/zacharyburnett/astropy-benchmarks.git --single-branch
```

Comment:
If you undo this line, I think we can then bring it out of draft and get it in. Thanks!